views/noscript-compiler.pug

```diff
@@ -0,0 +1,39 @@
+block content
```

Comment:
can we change these to `views/noscript/xyz.pug` instead of `views/noscript-xyz.pug` to cut down on clutter in the views directory?

Reply:
makes sense, sure